src/PowerSystems.jl

```diff
@@ -371,6 +371,8 @@ import InfrastructureSystems:
     get_time_series_names,
     get_next_time_series_array!,
     get_next_time,
+    get_units_info,
+    set_units_info!,
```

Comment:
Calling `set_units_info!` is potentially dangerous because of our implementation. What's your thinking behind exporting these to users? Do they need to be able to call them?